runtime/doc/news.txt

```diff
@@ -70,7 +70,24 @@ HIGHLIGHTS
 
 LSP
 
-• todo
+• `root_markers` in |vim.lsp.Config| can now specify markers priority.
+  Defaults to directory-up first search, which is more inline with
+  `nvim-lspconfig` but changes the default behaviour, which looked for every
+  marker before going direcotory-up.
```

Comment:
It's not strictly speaking true that each marker can "specify its priority", they are simply ordered by priority. Also there's no reason to mention nvim-lspconfig here.

```suggestion
• `root_markers` in |vim.lsp.Config| can now be ordered by priority.
```